spec/integration/composite_primary_key_spec.rb

```diff
@@ -3,26 +3,150 @@
 require "spec_helper"
 
 describe "composite_primary_key" do
-  context "without associations" do
-    with_model :CompositePrimaryKeyModel do
-      table primary_key: [:prefix, :postfix] do |t|
-        t.string :prefix
-        t.string :postfix
-        t.string :name
+  context 'without relations' do
```

Comment:
I think it'd be good to call these "associations" instead of "relations" because that's the Rails term for them.

["Relation"](https://www.postgresql.org/docs/16/glossary.html#GLOSSARY-RELATION) has a different meaning in PostgreSQL.

https://guides.rubyonrails.org/association_basics.html

Reply:
[55834f9](https://github.com/Casecommons/pg_search/pull/537/commits/55834f99d9e60fca7104332d459848cacba2a742)